pkg/primaza/constants/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2023 The Primaza Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package constants contains Primaza's constants
+package constants
```

Comment:
No much going on here, why do we need multiple constants files?

Reply:
the `doc.go` is one of the suggested approaches for [documenting packages](https://go.dev/blog/godoc)